tests/integration/test_authd/test_authd_use_password_invalid.py

```diff
@@ -0,0 +1,150 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: These tests will check invalid values in the authd.pass (for now just checks 'empty')
+       raises the expected error logs.
+
+components:
+    - authd
+
+suite: use_password
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-authd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+tags:
+    - enrollment
+    - authd
+'''
+
+import pytest
+
+from pathlib import Path
+
+from wazuh_testing.modules.authd import event_monitor as evm
+from wazuh_testing.tools import DEFAUL_AUTHD_PASS_PATH
+from wazuh_testing.tools.file import write_file, delete_file
+from wazuh_testing.tools.configuration import get_test_cases_data, load_configuration_template
+from wazuh_testing.tools.services import control_service
+
+
+# Constants & base paths
+TEST_NAME = Path(__file__).stem.replace('test_', '')
+DATA_PATH = Path(Path(__file__).parent, 'data')
+TESTS_CASES_PATH = Path(DATA_PATH, 'test_cases')
+CONFIGS_PATH = Path(DATA_PATH, 'config_templates')
+
+# Marks
+pytestmark = [pytest.mark.server, pytest.mark.tier(level=1)]
+
+# Paths
+cases_path = Path(TESTS_CASES_PATH, f"cases_{TEST_NAME}.yaml")
+config_path = Path(CONFIGS_PATH, f"config_{TEST_NAME}.yaml")
```

Comment:
```suggestion
# Reference paths
TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')

# Marks
pytestmark = [pytest.mark.server, pytest.mark.tier(level=1)]

# Configuration and cases data
test_cases_path = os.path.join(TEST_CASES_PATH, 'cases_authd_use_password_invalid.yaml')
configurations_path = os.path.join(CONFIGURATIONS_PATH, 'config_authd_use_password_invalid.yaml')

```

Reply:
Why replace the `TEST_NAME` constant with a hardcoded yaml name? I thought it would be better to force a yaml naming convention.

Reply:
For now, the focus is to apply the hardcoded settings. In this case, it is only a matter of respecting the proposed structure. In the future, we will improve this.

Reply:
Great! fixed on commit [`83234c3`](https://github.com/wazuh/wazuh-qa/pull/3721/commits/83234c3a0a59f9cfe2eca42f9e497d3e1896bf31)